common/changes/@microsoft.azure/openapi-validator-rulesets/tejaswis-DeleteRuleRPC-Put-V1-11_2023-08-07-17-46.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft.azure/openapi-validator-rulesets",
+      "comment": "Deleted files and context related to rule RPC-Put-V1-11",
+      "type": "patch"
+    }
+  ],
+  "packageName": "@microsoft.azure/openapi-validator-rulesets"
+}
```

Comment:
This change log is in appropriate

Reply:
Removed this file, my local VS code is behaving abnormal.